src/CommonBids.hs

```diff
@@ -13,7 +13,9 @@ module CommonBids(
 , takeoutDouble
 ) where
 
-import Auction(Action, constrain, define, forbid, pointRange, balancedHand, makeCall, makePass, suitLength, minSuitLength, maxSuitLength)
+import Auction(Action, constrain, define, forbid, pointRange, balancedHand,
+               makeCall, makeAlertableCall, makePass, suitLength,
+               minSuitLength, maxSuitLength)
```

Comment:
There's probably some auto-formatter out there that does these things properly. Next PR, look into that.

Reply:
A possible lead: https://hackage.haskell.org/package/hindent

Reply:
or the other options at https://mmhaskell.com/blog/2018/8/6/keeping-it-clean-haskell-code-formatters